### DISCUSSION TITLE
My benefits is not active

### DISCUSSION THREAD

**Discussion by Khushal942** [community/community#64851] [GitHub Education]
### Hi everyone,

I apply student pack in 23 date but my benefits is not active. at this time i see this massage "We're still processing your application. Once this is complete you will have access to the student developer pack offers." Please active my student benefits.

**Comment by arafajs** [comment]
It may take up to 72 hours for your benefits to be applied after your application is approved.

**Comment by Jaber-Saed** [comment]
Hello,

I understand that you've applied for the student pack on the 23rd, but your benefits have not yet been activated. You're currently seeing a message that your application is being processed and that you'll gain access to the student developer pack offers once this process is complete. I can imagine the anticipation, as access to these benefits can be really exciting for your development journey.

It's important to note that the processing time for student pack applications might vary based on the platform's procedures and current workload. Generally, the process involves verifying your student status, and this can take a bit of time to ensure accuracy.

If it's been a while and you haven't seen any changes, you might consider reaching out to the platform's support or help center. They should be able to provide you with more information on the status of your application and an estimated timeframe for when your benefits will be activated.

While waiting, you could continue working on your projects and preparing for the benefits you'll gain access to. Keep an eye on your email or notifications for updates on your application status.

I hope your student benefits are activated soon, and you can start enjoying the perks of the student developer pack!
